scripts/visualize/visualize-doy.R

```diff
@@ -6,44 +6,47 @@ visualize.doy <- function(viz = getContentInfo(viz.id = "doy-NM")){
     
   daily <- readData(viz[["depends"]][["daily"]])
   zoomer <- readData(viz[["depends"]][["zoomer"]])
+  watermark <- readData(viz[["depends"]][['watermark']])
+  
   zoomer.xml <- read_xml(zoomer)
   zoom.vb <- xml_attr(zoomer.xml, "viewBox")
   zoom.vb <- as.numeric(strsplit(zoom.vb, ' ')[[1]])
   xml_attr(zoomer.xml, "viewBox") <- NULL
   
   height <- viz[["height"]]
   width <- viz[["width"]]
+  yMax <- max(daily$Flow, na.rm = TRUE)
   
   doy_svg = svglite::xmlSVG({
-    par(omi=c(0,0,0,0), mai=c(0.5,0.75,0,0),las=1, xaxs = "i")
+    par(omi=c(0,0,0,0), mai=c(0.5,0.55,0,0),las=1, xaxs = "i")
```

Comment:
@jread-usgs I'm liking the idea of sourcing a script with global R objects. The `par` values would be something for sure that would be handy.